common/src/main/java/com/wynntils/screens/settings/widgets/TextInputBoxSettingsWidget.java

```diff
@@ -28,7 +28,7 @@ protected void onUpdate(String text) {
         Object parsedValue = configHolder.tryParseStringValue(text);
         if (parsedValue != null) {
             if (!parsedValue.equals(configHolder.getValue())) {
-                configHolder.setValue(parsedValue);
+                ((ConfigHolder<Object>) configHolder).setValue(parsedValue);
             }
```

Comment:
The problem here is that we have subclassed TextInputBoxSettingsWidget with CustomColorSettingsButton, which has another actual type in the ConfigHolder (CustomColor instead of String). Due to this, I cannot type the genetic paramenter to String, and that complicates this code.

Reply:
Actually, I should probably just generify TextInputBoxSettingsWidget...